docs/_markdown/ark.analysis.md

```diff
@@ -0,0 +1,36 @@
+analysis
+====================
+
+These libraries are like ground zero. They are the base libraries to use for analysis.
+
+ark.analysis.dimensionality\_reduction
+---------------------------------------------
+
+.. automodule:: ark.analysis.dimensionality_reduction
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+ark.analysis.spatial\_analysis
+-------------------------------------
+
+.. automodule:: ark.analysis.spatial_analysis
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+ark.analysis.visualize
+-----------------------------
+
+.. automodule:: ark.analysis.visualize
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+Module contents
+---------------
+
+.. automodule:: ark.analysis
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
Can this be deleted or reformatted?  It shows up below the other subpackages as if it were a subpackage.

Reply:
Deleted for the most recent commit.